lisa/sut_orchestrator/azure/features.py

```diff
@@ -668,6 +669,67 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
             len(all_nics) - self.origin_extra_synthetic_nics_count - 1
         )
 
+    def create_route_table(
+        self,
+        nic_name: str,
+        route_name: str,
+        subnet_mask: str,
+        dest_hop: str,
+        em_first_hop: str = "",
+        next_hop_type: str = "",
+    ) -> None:
+        # some quick checks that the subnet mask looks correct
+        assert_that(subnet_mask).described_as(
+            "subnet mask should be prefix format X.X.X.X/YY"
+        ).contains("/")
+        assert_that(subnet_mask.split("/")).described_as(
+            "subnet mask should be prefix format X.X.X.X/YY"
+        ).is_length(2)
```

Comment:
The assert_that is a chained validation. so you can put two checks in the same line, instead of write twice. But use regexp is more comprehensive check.

Reply:
let's do regex